packages/jest-util/src/__tests__/stacktrace-test.js

```diff
@@ -24,18 +24,11 @@ const windowsStackTrace = `  ` +
   at attemptAsync (..\\jest-jasmine2\\vendor\\jasmine-2.4.1.js:1919:24)`;
 
 it('should exclude jasmine from stack trace for windows paths', () => {
-  const messages = formatResultsErrors(
-    [
-      {
-        ancestorTitles: [],
-        failureMessages: [windowsStackTrace],
-        title: 'windows test',
-      },
-    ],
-    {
-      rootDir: '',
-    },
-  );
+  const messages = windowsStackTrace.split('\n');
 
-  expect(messages).toMatchSnapshot();
+  expect(
+    messages
+      .filter((filename, index) => !toDiscard(filename, index))
```

Comment:
It seems to me like this is not the correct test as you are basically re-implementing the prepare stack trace function. The proper way to test this component is to make sure the `Error.prepareStackTrace` function does the correct thing. This can be achieved by calling `setPrepareStackTrace` on a fake error object, then invoking it with mock data and ensuring that the stack trace it generates is what we expect.